integrations-service/integrations/utils/integrations/remote_browser.py

```diff
@@ -104,26 +104,26 @@ async def _reset_mouse(self) -> None:
             window.$$julep$$_initialized = true;
             """)
 
-    @staticmethod
-    def _with_error_and_screenshot(f):
-        @wraps(f)
-        async def wrapper(self: "PlaywrightActions", *args, **kwargs):
-            try:
-                result: RemoteBrowserOutput = await f(self, *args, **kwargs)
-                await self._wait_for_load()
+    # @staticmethod
```

Comment:
What’s this commented out code?

Reply:
@Ahmad-mtos Are you certain this works without a problem?

Reply:
yes, I left it commented out in case we made it optional moving forward.